packages/cli/bin/merkur.mjs

```diff
@@ -28,6 +31,36 @@ const inspectOption = new Option('--inspect', 'Debugging widget server');
 const verboseOption = new Option('--verbose', 'Verbose mode which show debug information.');
 const analyzeOption = new Option('--analyze', 'Analyze bundle.');
 
+let customCommands = [];
+
+// Extend commands from directory
+const extendCommandsFromDir = async commandsDir => {
+  customCommands = customCommands.concat(
+    fs.readdirSync(commandsDir)
+      .map(command => ({ dir: commandsDir, command }))
+      .filter(
+        ({ command, dir }) => !customCommands.some(({ command: existingCommand }) => existingCommand === command) &&
+        fs.statSync(path.join(dir, command)).isFile() &&
+        (command.endsWith('.js') || command.endsWith('.mjs'))
+      )
+  );
+};
+
+// Get merkur custom commands
+const merkurDir = path.resolve(process.cwd(), 'node_modules/@merkur');
+if (fs.existsSync(merkurDir)) {
+    let dirs = [];
```

Comment:
Why are we setting a default value here and right on the next line we assign a new value? Can't we just assign the new value directly?

Reply:
New value assign directly.